tree/master/packages/extension/public/locales/pl/translation.json

```
@@ -0,0 +1,162 @@
+{
+	  "Copied": "Skopiowane",
+	  "<unknown>": "<nieznane>",
+	  "click to select or drag and drop the file here": "kliknij, aby wybrać lub przeciągnij i upuść plik tutaj",
+	  "{{name}} ({{size}} bytes)": "{{name}} ({{size}} bajtów)",
+	  "... loading ...": "... Ładowanie ...",
+	  "Generated 12-word mnemonic seed:": "Wygenerowane 12-wyrazowe ziarno mnemoniczne:",
+	  "Copy to clipboard": "Skopiuj do schowka",
+	  "Back": "Z powrotem",
+	  "Create new account (root or derived)": "Utwórz nowe konto (główne lub pochodne)",
+	  "Create new account": "Stwórz nowe konto",
+	  "Import account from pre-existing seed": "Importu konto z wcześniej istniejącego materiału siewnego",
+	  "Restore account from backup JSON file": "Przywróć konto z zapasowego pliku JSON",
+	  "Attach external QR-signer account": "Dołącz zewnętrzne konto osoby podpisującej kod QR",
+	  "External QR accounts and Access": "Zewnętrzne konta QR i dostęp",
+	  "Allow Camera Access": "Zezwól na dostęp do kamery",
+	  "Display address format for": "Wyświetl format adresu dla",
+	  "Language": "Język",
+	  "Open extension in new window": "Otwórz wtyczke w nowym oknie",
+	  "Account name is too short": "Nazwa konta jest za krótka",
+	  "A descriptive name for your account": "Opisowa nazwa twojego konta",
+	  "Password is too short": "Hasło jest za krótkie",
+	  "Passwords do not match": "Hasła nie pasują do siebie",
+	  "A new password for this account": "Nowe hasło do tego konta",
+	  "Repeat password for verification": "Powtórz hasło do weryfikacji",
+	  "Rename": "Zmień nazwę",
+	  "Derive New Account": "Wyprowadź nowe konto",
+	  "Export Account": "Eksportuj Konto",
+	  "Forget Account": "Zapomnij konto",
+	  "Visible (always inject)": "Widoczny (zawsze wstrzykiwany)",
+	  "You currently don't have any accounts. Create your first account to get started.": "Obecnie nie masz żadnych kont. Aby rozpocząć, utwórz swoje pierwsze konto.",
+	  "Accounts": "Konta",
+	  "Authorize": "Autoryzuj",
+	  "Only approve this request if you trust the application. Approving gives the application access to the addresses of your accounts.": "Zatwierdź tę prośbę tylko wtedy, gdy ufasz aplikacji. Zatwierdzenie daje aplikacji dostęp do adresów Twoich kont.",
+	  "Yes, allow this application access": "Tak, zezwól tej aplikacji na dostęp",
+	  "Add the account with the generated seed": "Dodaj konto z wygenerowanym seedem",
+	  "Create an account": "Utwórz konto",
+	  "Please write down your wallet's mnemonic seed and keep it in a safe place. The mnemonic can be used to restore your wallet. Keep it carefully to not lose your assets.": "Proszę zapisać ziarno mnemoniczne portfela i przechowywać je w bezpiecznym miejscu. Mnemonik może być użyty do przywrócenia portfela. Przechowuj go ostrożnie, aby nie stracić swoich aktywów.",
+	  "I have saved my mnemonic seed safely.": "Bezpiecznie zapisałem moje ziarno mnemoniczne.",
+	  "Next step": "Następny krok",
+	  "Derivation Path": "Ścieżka Wyprowadzenia",
```

```suggestion
	  "Derivation Path": "Ścieżka pochodna",
```
aby trzymać się konswekwetnie "pochodnego konta".